src/cpp/lpnamer/problem_modifier/AntaresProblemToXpansionProblemTranslator.cpp

```diff
@@ -18,17 +20,37 @@ AntaresProblemToXpansionProblemTranslator::translateToXpansionProblem(
   problem->init();
```

Comment:
Is it still necessary as we do `ret->init()` in `create_solver` ? The remark is valid for everywhere else in the code we call this function